exo/inference/tinygrad/inference.py

```
@@ -41,8 +41,19 @@
 
 def build_transformer(model_path: Path, shard: Shard, model_size="8B", device=None):
   # build model
+  try:
+    with open(model_path/"config.json", "r") as f:
+      config = json.load(f)
+
+  except FileNotFoundError:
+    raise Exception(f"Config file not found in {model_path}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)